code/game/machinery/doors/door.dm

```diff
@@ -221,7 +221,8 @@
 	return
 
 /obj/machinery/door/attackby(obj/item/I, mob/living/user, params)
-	if(!user.combat_mode && (I.tool_behaviour == TOOL_CROWBAR || istype(I, /obj/item/fireaxe)))
+	var/list/modifiers = params2list(click_parameters)
```

Comment:
sir did you test this

![image](https://user-images.githubusercontent.com/35135081/108041744-d23ed000-6ff3-11eb-883c-2f07530ff0c6.png)

Reply:
surely i did